style.css

```diff
@@ -871,4 +1040,71 @@ textarea::-webkit-input-placeholder {
   .portfolio {
     height: 448px;
   }
+
+  .modal {
+    min-width: 1156px;
+    height: 1163px;
+    top: 5%;
+    left: 4%;
+    right: 4%;
+    max-height: calc(100vh - 100px);
+    overflow-y: auto;
+  }
+
+  .modal-body {
+    margin-left: 42px;
+  }
+
+  .modal-body .portfolio {
+    min-width: 1108px;
+    height: 586px;
+  }
+
+  .project-wrapper {
+    display: flex;
+  }
+
+  .description {
+    min-width: 800px;
+    height: 382px;
+    margin-right: 20px;
+  }
+
+  .modal-body button {
+    width: 124px;
+  }
+
+  .wrap {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 16px;
+  }
+
+  .modal-header {
+    display: flex;
+  }
+}
+
+@media screen and (min-width: 1400px) {
+  .modal {
+    min-width: 1350px;
+    left: 8%;
+    right: 8%;
+  }
+
+  .modal-body {
+    margin-left: 0;
+  }
+
+  .modal-body .portfolio {
+    min-width: 1300px;
+    height: 700px;
+  }
+
+  .description {
+    min-width: 1000px;
+    height: 382px;
+    margin-right: 20px;
+  }
 }
```

Comment:
- [ ] As stated the project requirement Kindly make sure that the **popup is responsive, it fits  desktop sizes properly.** .So Kindly Fix it.

Your Project Popup Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/217580262-d15c3e6b-4a94-46ed-a1c5-c5265989cedb.PNG)

The Figma Design Popup Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/217580600-8de52323-cfab-4172-99b0-5936ef6d7f14.PNG)